src/architecture/services.md

```diff
@@ -131,19 +148,22 @@ cryptocurrency service persists account balances, which are changed by transfer
 and issuance transactions.
 
 Exonum persists blockchain state in a global key-value storage implemented
-using [LevelDB][leveldb]. Each service needs to define a set of data collections
+using [LevelDB][leveldb] and [RocksDB][rocksdb]. Each service needs to define a
```

Comment:
Nitpick: This may sound like *both* LevelDB and RocksDB are needed for the KVS implementation, which is not true. Maybe change to "...storage implemented with one of supported backends (LevelDB and RocksDB as of Exonum 0.2)"?